### ISSUE TITLE
Another issue for tab key users on the map page while navigating through the mode-context

### ISSUE THREAD

**Issue by Ifycode** [ushahidi/platform#4116]
**Expected behaviour:**
When using tab key to navigate to the elements in this particular component (tagged mode-context in the code base), users expect that there is visible outline around the elements to know where they are currently on.

**Actual behaviour:**
Nothing shows that the user is currently on a particular element when they navigate this portion using the tab key which may cause confusion. Example, in the screenshot below, only the header that contains my name and the select language select option has visible outline when tabbing through this area. After the header, there is no sign that anything is happening until you get to select language option.

![mode-context](https://user-images.githubusercontent.com/45185388/95983505-9215d980-0dd6-11eb-8904-2902817c3029.PNG)

**Some additional findings:**
- Some elements can't be accessed by tab users e.g. the tiny button after the text "Basic post" is skipped.
- Another example is that the check box just before the text "Basic post" is still hidden on selecting it with the tab and space key. I suggest that the styles applied on hover should be applied here too so that it's visible to the user when they are communicating with every affected portion in the mode-context.

**Steps to reproduce behaviour:**
- Log in to your account
- Use tab key to navigate through the part shown in the screenshot above

Aha! Link: https://ushahiditeam.aha.io/features/PROD-868

**Comment by Ifycode** [comment]
Hi, @Angamanga @rowasc found this issue and it's solution.

**Comment by Angamanga** [comment]
@Ifycode Well done! Please go ahead and work on the issue!

**Comment by Ifycode** [comment]
Thank You! @Angamanga

**Comment by siggrid** [comment]
@Angamanga @Ifycode Hi! I’ve created an issue about the same problem with possible solution.

**Comment by Angamanga** [comment]
@siggrid Ah, that is true! @Ifycode You may want to hold back before doing work on this.

**Comment by Ifycode** [comment]
Oh! ok @Angamanga although I already updated the issue with my additional findings

**Comment by Angamanga** [comment]
@Ifycode For Outreachy though, I think it is ok to have submitted prs but not got them merged to get them counted. I don't recommend that in normal cases since it would mean a lot of "unused" work, but if you already started working on it, go ahead and submit a pr!

**Comment by Ifycode** [comment]
Ok. Thank you @Angamanga Ok, for the purpose of outreachy, I will submit my PR. But I will like to know more about the normal cases as you have mentioned.

**Comment by Angamanga** [comment]
@Ifycode With normal cases I meant I try not to assign more than one person to the same issue. Don't worry, you are already doing it right by submitting issues and prs :)

**Comment by Ifycode** [comment]
Oh. OK. I understand what you mean now @Angamanga

**Comment by Ifycode** [comment]
@Angamanga 

> @Angamanga The thing is I only edited the code for issue #4115 #4116 in the dev tool and not in the actual code base. I found that some of the Issues also happens on the map and search area of the page. I suggest that since @siggrid found the other issue first, if she has started work on it, then let me work on this issue I just raised instead.

_Originally posted by @Ifycode in https://github.com/ushahidi/platform/issues/4119#issuecomment-708659017_
